x-pack/plugins/observability_solution/inventory/server/routes/entities/get_latest_entities.ts

```diff
@@ -47,14 +35,13 @@ export async function getLatestEntities({
      | ${getEntityDefinitionIdWhereClause()}
      | SORT ${sortField} ${sortDirection}
      | LIMIT ${MAX_NUMBER_OF_ENTITIES}
-     | KEEP ${ENTITY_LAST_SEEN}, ${ENTITY_TYPE}, ${ENTITY_DISPLAY_NAME}, ${ENTITY_ID}
```

Comment:
Hi @cauemarcondes, 

When merging this code into my current working branch, I noticed that removing this is causing all three entity type attributes to be set to `null` in our API response.

Was this the intended behavior?

![Screenshot 2024-10-01 at 08 45 11](https://github.com/user-attachments/assets/0e2c1210-897c-416a-b5ad-f0794b4bc416)

Reply:
yes, since we don't know what metadata is available per entity definition we need to get it all.